settings.py

```diff
@@ -80,7 +80,8 @@
 CSP_CONNECT_SRC = ("'self'", 'https://gtexportal.org', 'https://www.google-analytics.com', 'https://igv.org',
                    'https://storage.googleapis.com', 'https://s3.amazonaws.com', 'https://igv-genepattern-org.s3.amazonaws.com', 'https://hgdownload.soe.ucsc.edu',  # used by IGV
                    'https://reg.genome.network')
-CSP_SCRIPT_SRC = ("'self'", "'unsafe-eval'", 'https://www.googletagmanager.com')
+CSP_SCRIPT_SRC = ("'self'", "'unsafe-eval'", 'https://www.googletagmanager.com',
+                  "'sha256-A16xkExeIj9J9P70pOQ+HDBXdFYcm2Cn3J/phdqk9hc='")  # luigi UI script
```

Comment:
where does this value come from?

Reply:
This is the hashed encoding of the actual script that the luigi UI is executing. The way CSP blocking works (at least in our local mode) it will not execute a script not allowed by the csp script policy, but the error it generates actually says what you would need to change to allow it, including generating what the hash would be if we wanted to use that so I just copied it from there